satellite/gracefulexit/pathcollector.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit
+
+import (
+	"context"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/metainfo"
+)
+
+var _ metainfo.Observer = (*PathCollector)(nil)
+
+// PathCollector uses the metainfo loop to add paths to node reservoirs
+//
+// architecture: Observer
+type PathCollector struct {
+	db              DB
+	metainfoService *metainfo.Service
```

Comment:
I don't see this service being used anywhere in this file. Am I missing something here?

Reply:
Good catch.  I had some code that was using it, but it was removed and I forgot to remove the metainfoService.